package.json

```
@@ -25,114 +25,112 @@
     "prepare": "./scripts/link-them.sh"
   },
   "dependencies": {
-    "@glimmer/syntax": "^0.42.0",
+    "@glimmer/syntax": "^0.42.2",
     "broccoli-bridge": "^1.0.0",
     "broccoli-caching-writer": "^3.0.3",
     "broccoli-debug": "^0.6.4",
     "broccoli-filter": "^1.2.4",
     "broccoli-funnel": "^2.0.2",
     "broccoli-merge-trees": "^3.0.1",
     "broccoli-persistent-filter": "^2.3.1",
-    "broccoli-plugin": "^1.3.1",
-    "broccoli-source": "^1.1.0",
-    "broccoli-stew": "^2.0.0",
+    "broccoli-plugin": "^3.0.0",
```

May I suggest a change here as an addon user?

```suggestion
    "broccoli-plugin": "1.1.0 - 3",
```

If you specify a more permissive range in the dependencies, the consuming apps will be able to use either v1, v2 or v3. This will allow their dependencies to be deduplicated. This would be especially useful for those who use something like ember-dependency-lint (that makes your CI fail when your app has multiple versions of a same dependency).

This remark doesn’t concern the devDependencies as they are neither downloaded nor loaded by the consuming apps.

---

Thanks for the catch. Just pushed a commit to revert to the original code here.